.github/workflows/master.yaml

```diff
@@ -0,0 +1,57 @@
+name: Dev Packages and Docker images
+on:
+  push:
+    branches: [master]
+
+# Checks if any concurrent jobs under the same pull request or branch are being executed
+concurrency:
+  group: ci-master
+  cancel-in-progress: true  
+
+jobs:
+  build-dev-packages:
+    uses: falcosecurity/falco/.github/workflows/reusable_build_packages.yaml@main
+    with:
+      arch: x86_64
+    secrets: inherit
+  
+  build-dev-packages-arm64:
+    uses: falcosecurity/falco/.github/workflows/reusable_build_packages.yaml@main
+    with:
+      arch: aarch64
+    secrets: inherit
+    
+  publish-dev-packages:
+    needs: [build-dev-packages, build-dev-packages-arm64]
+    uses: falcosecurity/falco/.github/workflows/reusable_publish_packages.yaml@main
+    with:
+      bucket: '-dev'
+      version: ${{ needs.build-dev-packages.outputs.version }}
+    secrets: inherit
+    
+  build-dev-docker:
```

Comment:
Both `build-dev-docker` and its arm64 counterpart require `build-dev-packages` because they use its output.